### DISCUSSION TITLE
Builds fail now if the latest git tag is not a semver version

### DISCUSSION THREAD

**Discussion by SKoschnicke** [goreleaser/goreleaser#2534] [Ideas, Issue Triage, and General discussions]
With https://github.com/goreleaser/goreleaser/pull/2503 it is now not possible anymore to do builds with goreleaser if the latest git tag is not a semantic version tag.

At least in our repository there are other tags and the build is triggered in CI on every push to master. But now it fails mostly. Is it intended that this use case is not supported, or am I missing something?

This also applies to snapshot builds because the tag is validated before the transformation specified in `snapshot.name_template`.

I totally understand that it leads to strange errors later and adds complexity if the version is not semver compatible, but at least for us it would be very useful to have a flag to intentionally disable this check.

**Comment by caarlos0** [comment]
Hi, I think you can specify `GORELEASER_CURRENT_TAG` env to go around that, can you try?

Related docs pages:
- https://goreleaser.com/limitations/semver/
- https://goreleaser.com/cookbooks/set-a-custom-git-tag/
